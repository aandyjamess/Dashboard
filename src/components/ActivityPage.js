import React from "react";
import PropTypes from "prop-types";

import * as util from "../util";
import * as copy from "./../copy";

import { ActivityList, Group } from "./index";

const ActivityPage = ({actions, receipts}) => (
  <main className="col-md-12" >
    <h6 className="underlined"> Activity </h6>
    <h5> Activity </h5>
    <p> {copy.activity} </p>
    <Group withFn={util.sameDay} list={receipts} Component={ActivityList} props={{actions}} />
  </main>
);

ActivityPage.PropTypes = {
  receipts: PropTypes.Array
};

ActivityPage.defaultProps = {
  receipts: []
};

export default ActivityPage;
