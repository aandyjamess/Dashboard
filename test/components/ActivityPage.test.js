import React from "react";
import toJson from "enzyme-to-json";

import { shallow } from "enzyme";

import ActivityPage from "../../src/components/ActivityPage";

describe("ActivityPage", () => {
  it("renders correctly", () => {
    expect(toJson(shallow(<ActivityPage />))).toMatchSnapshot();
  });
});

