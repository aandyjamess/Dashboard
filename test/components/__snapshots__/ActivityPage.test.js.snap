// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityPage renders correctly 1`] = `
<main
  className="col-md-12"
>
  <h6
    className="underlined"
  >
     Activity 
  </h6>
  <h5>
     Activity 
  </h5>
  <p>
     
    See a record of everyone you have shared details with.
     
  </p>
  <Group
    Component={[Function]}
    list={Array []}
    props={
      Object {
        "actions": undefined,
      }
    }
    withFn={[Function]}
  />
</main>
`;
